src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -154,8 +276,64 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityProps> = ({
         .times(isAsymAssetSide || isAsymRuneSide ? 2 : 1)
         .toFixed(),
     )
+    setCrytoAssetBalance(underlyingAssetAmountCryptoPrecision)
+    setFiatAssetBalance(underlyingAssetValueFiatUserCurrency)
+    setCrytoRuneBalance(underlyingRuneAmountCryptoPrecision)
+    setFiatRuneBalance(underlyingRuneValueFiatUserCurrency)
   }, [isAsymAssetSide, isAsymRuneSide, percentageSelection, userlpData])
 
+  // We reuse lending utils here since all this does is estimating fees for a given deposit amount with a memo
+  // It's not going to be 100% accurate for EVM chains as it doesn't calculate the cost of depositWithExpiry, but rather a simple send,
+  // however that's fine for now until accurate fees estimation is implemented
+  const { data: estimatedRuneFeesData, isLoading: isEstimatedRuneFeesDataLoading } =
+    useQuoteEstimatedFeesQuery({
+      collateralAssetId: thorchainAssetId,
+      collateralAccountId: runeAccountId ?? '', // fixme
+      repaymentAccountId: runeAccountId ?? '', // fixme
+      repaymentAsset: runeAsset ?? null,
+      repaymentAmountCryptoPrecision: actualRuneCryptoLiquidityAmount,
+      confirmedQuote,
+    })
+
+  const { data: estimatedPoolAssetFeesData, isLoading: isEstimatedPoolAssetFeesDataLoading } =
+    useQuoteEstimatedFeesQuery({
+      collateralAssetId: poolAsset?.assetId ?? '', // fixme
```

Comment:
are these fixmes still needed?

Reply:
They are not - improvement comments in https://github.com/shapeshift/web/pull/6175/commits/f8bb0b3f9803b822b907745861b5cb318940a4ab to explain why they might be `undefined`.